### PR TITLE
Re-ordering accordion. Done without recreating items

### PR DESCRIPTION
**Pull request by pmbohm**

### CODE REVIEW

**Comment by dnahodil** [comment]
I'll review this. However, it looks like it requires a rebase, @pmbohm.

File: web-app/css/imosTheme.css [hunk 1]
```diff
@@ -337,5 +337,14 @@ ul.x-tab-strip-top {
...
 
+.subsetPanelAccordionItem .x-tool:not(:first-of-type) {
+    margin: 3px 0;
+}
+
+.subsetPanelAccordionItem .x-tool:first-of-type {
+    margin-left: 6px;
+    margin-right: 0;
+}
+
```
Comment (dnahodil) [inline web-app/css/imosTheme.css]: Are these changes really IMOS-theme specific? or should they go in the general.css file?

File: web-app/js/portal/details/NcWmsPanel.js [hunk 2]
```diff
@@ -323,7 +323,6 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
         if (this.geoNetworkRecord) {
             this._addDateTimeFilterToLayer();
             this.geoNetworkRecord.updateNcwmsParams(dateRangeStart, dateRangeEnd, geometry);
-            Ext.MsgBus.publish(PORTAL_EVENTS.DATA_COLLECTION_MODIFIED);
```
Comment (dnahodil) [inline web-app/js/portal/details/NcWmsPanel.js]: Why don't we send the modded event when filters are changes anymore?

File: web-app/js/portal/details/SubsetItemsWrapperPanel.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+        var tabPanelForLayer = new Portal.details.SubsetItemsTabPanel({
+            map: cfg.map,
+            layer: cfg.layer,
+            listeners: {
+                beforeTabChange: this._doTracking
+            }
+        });
+
+        this.layer = cfg.layer;
```
Comment (dnahodil) [inline web-app/js/portal/details/SubsetItemsWrapperPanel.js]: This line shouldn't be needed. All values from config should be automatically applied to 'this'.

**Comment by dnahodil** [comment]
It also seems to introduce 2 bugs:
1) If you ctrl-click add a few layers quickly then their subsetting options can fail to load
2) Rearranging the layers in the accordion does not change the order they appear on the map

File: web-app/css/imosTheme.css (hunk 1 above)
Comment (pmbohm) [inline web-app/css/imosTheme.css]: mmm... My thoughts are that themeing is useless as everyone starts with the imosTheme anyway.
This change affect prettiness only and as such why not keep it for IMOS

File: web-app/js/portal/details/SubsetItemsWrapperPanel.js (hunk 3 above)
Comment (pmbohm) [inline web-app/js/portal/details/SubsetItemsWrapperPanel.js]: Absolutely!

File: web-app/css/imosTheme.css (hunk 1 above)
Comment (dnahodil) [inline web-app/css/imosTheme.css]: Okay, cool. I didn't look into how these changes affect things, but if they're not required for the thing to work then they're fine in here as you say.

File: web-app/js/portal/details/NcWmsPanel.js (hunk 2 above)
Comment (pmbohm) [inline web-app/js/portal/details/NcWmsPanel.js]: This is now a while ago. It was the cause of an infinite loop in my changes. As to why it was needed in the first place ... no idea.
Basically now the ActiveGeoNetworkRecordStore triggers this event
the DownloadPanel and SubbsettingPanel listen
So the event is more like AN_ITEM_IN_THE_DATA_COLLECTION_STORE_IS_MODIFIED_OR_JUST_THE_ORDER

**Comment by pmbohm** [comment]
@dnahodil bugs now fixed

**Comment by dnahodil** [comment]
Changes look good @pmbohm. Nice one.

**Pull request merged**